src/codegen/codegen.test.ts

```diff
@@ -95,7 +95,7 @@ describe('Code generation', () => {
               cloud: {
                 loadZones: {
                   distribution: 'even',
-                  loadZones: [],
+                  zones: [],
```

Comment:
I found myself using `loadZones.loadZones` when working on codegen and decided to rename it to `loadZones.zones` instead.